Src/BootCamp.Chapter/Computer/ComputerCase.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace BootCamp.Chapter.Computer
+{
+    public class ComputerCase
+    {
+        private string _manufacturersId;
+        public ComputerCase(string manufacturersId)
+        {
+            _manufacturersId = manufacturersId;
+        }
+
+        public string GetManufacturersId()
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Consider making method 'GetManufacturersId' a property.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=5121609)